examples/winit.rs

```diff
@@ -7,14 +7,16 @@ fn main() {
     let event_loop = EventLoop::new();
     let window = WindowBuilder::new().build(&event_loop).unwrap();
 
-    let context = GlContext::create(&window, GlConfig::default()).unwrap();
+    let context = GlContext::create(&window, GlConfig::default(), None).unwrap();
```

Comment:
Worth noting that this is a breaking API change. It might be possible to revise this to not be API-breaking. (i.e. create() remains the same, add create_shared() that takes the extra param)